python/ml4ir/base/features/feature_fns/categorical.py

```diff
@@ -30,57 +30,56 @@ def categorical_embedding_with_hash_buckets(feature_tensor, feature_info):
         embedding_size: int; dimension size of the categorical embedding
         merge_mode: str; can be one of "mean", "sum", "concat" representing the mode of combining embeddings from each categorical embedding
     """
+    feature_layer_info = feature_info.get("feature_layer_info")
+    embeddings_list = list()
+    for i in range(feature_layer_info["args"]["num_hash_buckets"]):
+        augmented_string = layers.Lambda(lambda x: tf.add(x, str(i)))(feature_tensor)
 
-    # Numeric input features
-    if feature_info["dtype"] in (tf.float32, tf.int64):
-        raise NotImplementedError
```

Comment:
Don't we need the check anymore?

Reply:
It's going to fail regardless if it is not a string. When we support other types, we can add it back.